### PR TITLE
Refactor/auto compound upgrade

### PR DESCRIPTION
**Pull request by LeonardoVieira1630**
- [X] Change function name 
- [X] Add constant
- [X] Optimize reward distribution

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -529,7 +549,7 @@ impl Contract {
             msg,
             CONTRACT_ID_REF_EXC.parse().unwrap(), // contract account id
             1,                                    // yocto NEAR to attach
-            Gas(75_000_000_000_000),              // gas to attach
+            Gas(80_000_000_000_000),              // gas to attach 75
```
Comment (pmarangone) [inline src/lib.rs:552]: What "gas to attach 75" mean here?

File: src/lib.rs [hunk 2]
```diff
@@ -721,20 +741,23 @@ impl Contract {
...
 
         let residue: u128 = *self
             .last_reward_amount
-            .get(&"ref-finance.testnet@193#1".to_string())
+            .get(&"exchange.ref-dev.testnet@193#6".to_string())
             .unwrap();
-            
         self.last_reward_amount.insert(
-            "ref-finance.testnet@193#1".to_string(),
+            "exchange.ref-dev.testnet@193#6".to_string(),
```
Comment (pmarangone) [inline src/lib.rs:758]: Please, create a variable for this. It will make it easier to update it later

File: src/lib.rs (hunk 1 above)
Comment (LeonardoVieira1630) [reply src/lib.rs:552]: Nothing

File: src/lib.rs (hunk 2 above)
Comment (LeonardoVieira1630) [reply src/lib.rs:758]: I will. I am just waiting to do it in the task "Generalize vault contract", that is one of the next tasks that i have.

**Pull request merged**